bin/check_openmpvv.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+#
+#  check_openmpvv.sh
+#  Assumes run_openmpvv.sh has been executed.
+#
+# --- Start standard header to set AOMP environment variables ----
+realpath=`realpath $0`
+thisdir=`dirname $realpath`
+. $thisdir/aomp_common_vars
+# --- end standard header ----
+
+set -e
+
+pushd $AOMP_REPOS_TEST/$AOMP_OPENMPVV_REPO_NAME
+testname_regex='Test\sname":\s"(.*)"'
+compiler_regex='Compiler\sresult":\s"([A-Z]+)'
+runtime_regex='Runtime\sresult":\s"([A-Z]+)'
+function parse(){
+  local file=$2
+  while read -r line; do
+    local openmpvvtest=""
+    if [[ "$line" =~ $testname_regex ]]; then
+      openmpvvtest=${BASH_REMATCH[1]}
+    elif [[ "$line" =~ $compiler_regex ]]; then
+      compresult=${BASH_REMATCH[1]}
+    elif [[ "$line" =~ $runtime_regex ]]; then
+      runresult=${BASH_REMATCH[1]}
+    fi
+    if [ "$openmpvvtest" != "" ];then
+      results[$openmpvvtest]="$compresult"
+      results[$openmpvvtest]+="$runresult"
+      compresult=""
+      runresult=""
+    fi
+  done < "$file"
+}
+
+function checkstatus(){
+  tmppassfile="results_report$1/tmppassing-tests.txt"
+  tmpcompfailfile="results_report$1/tmpmake-fail.txt"
+  tmprunfailfile="results_report$1/tmpfailing-tests.txt"
+
+  local val=$2
+  if [ "$val" == "PASSPASS" ]; then
+    echo $i >> $tmppassfile
+  elif [ "$val" == "FAIL" ]; then
+    echo $i >> $tmpcompfailfile
+  elif [ "$val" == "PASSFAIL" ]; then
+    echo $i >> $tmprunfailfile
+  fi
+
+}
+
+function checkresult(){
+  passfile="results_report$1/passing-tests.txt"
```

Comment:
Can you add here and above a comment what `$1` contains, please?
IIUC it's the OpenMP version. is that right?

Reply:
AFAIK this should be correct, I'll re-run and check; then add a comment with the info and an example.